projects/app/src/__tests__/dictionary/dictionary.test.ts

```diff
@@ -152,7 +152,10 @@ void test(`convertPinyinWithToneNumberToToneMark`, () => {
     [`dou4`, `dòu`],
     [`dou5`, `dou`],
 
+    // A few examples
     [`hao3`, `hǎo`],
```

Comment:
```suggestion
    [`hao3`, `hǎo`],
    ['yi2', 'yí'],
    ['wo2', 'wó'],
    ['lu2', 'lú'],
    ['lü2', 'lǘ'],
```

Reply:
thanks for the contribution, but the PR is already in the queue merge so I can't commit this